<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskTrackerClass</class>
 <widget class="QWidget" name="TaskTrackerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskTracker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,2">
   <item>
    <widget class="QDateEdit" name="DateField"/>
   </item>
   <item>
    <widget class="QWidget" name="DurationWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="TimeLabel">
        <property name="text">
         <string>Time spent: 00:00:00.000</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Session duration (hours):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="SessionDurationBox">
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="TaskContainer"/>
   </item>
   <item>
    <widget class="QPushButton" name="StartButton">
     <property name="text">
      <string>Start session</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>StartButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>TaskTrackerClass</receiver>
   <slot>OnStartSessionPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnStartSessionPress()</slot>
 </slots>
</ui>
